PR_REVIEW_CHECKLIST.md

```diff
@@ -0,0 +1,98 @@
+# PR review checklist
+
+This document lists specific checks that should be done when reviewing a PR. This means these items should be checked by:
+1. Every contributor submitting a Pull Request
+2. Every C+ (Contributor Plus) team member reviewing a PR
+3. Every Expensify employee submitting / reviewing a PR
+
+This is a living document, so we will be updating this from time to time. Feel free to suggest additions or changes in the #expensify-open-source Slack channel.
+
+# Check PR template
+
+1. Make sure the PR template includes screenshots or videos of the changes being tested on **all platforms**. If there are one or more platforms missing, follow up with the contributor to determine why - we shouldn't be merging changes that aren't tested everywhere.
```

Comment:
Is this kind of redundant? Shouldn't we only put things into this file that are not covered in the PR template? If we change something about this later we have to update it in two places.

Reply:
Good idea, it's mentioned in the template AND in Contributing.md so i'll remove it from here